### PR TITLE
AM2R: Restream friendly tracker

### PR DESCRIPTION
**Pull request by JeffGainsNGames**
image

### CODE REVIEW

**Comment by JeffGainsNGames** [comment]
correct 5x5 tracker
image

File: randovania/data/gui_assets/tracker/game-images/am2r/missile_launcher.png
Comment (Miepee) [inline randovania/data/gui_assets/tracker/game-images/am2r/missile_launcher.png:1]: Including these sprites means you need to change the licensing attribution.

**Review by Miepee — CHANGES_REQUESTED** [review]

**Review by Miepee — CHANGES_REQUESTED** [review]
You also need to update the credits in the readme.

**Comment by JeffGainsNGames** [comment]
The readme already lists Shirty for Missile Launcher and Abyss for "other sprites". Are these statements not enough?

**Comment by Miepee** [comment]
Oh yeah whoops, my bad.

**Review by Miepee — APPROVED** [review]

**Pull request merged**